examples/domainmodel/src/cli/cli-util.ts

```diff
@@ -41,3 +45,13 @@ export function extractDocument(fileName: string, extensions: string[], services
 export function extractAstNode<T extends AstNode>(fileName: string, extensions: string[], services: LangiumServices): T {
     return extractDocument(fileName, extensions, services).parseResult?.value as T;
 }
+
+export const generateAction = (fileName: string, opts: GenerateOptions): void => {
+    const domainmodel = extractAstNode<Domainmodel>(fileName, languageMetaData.fileExtensions, createDomainModelServices());
+    const generatedDirPath = generateJava(domainmodel, fileName, opts.destination);
+    console.log(colors.green(`Java classes generated successfully: ${colors.yellow(generatedDirPath)}`));
+};
```

Comment:
This function shouldn't be here because that creates a cyclic dependency between `cli-util.ts` and `generator.ts`. Could you move it back to `index.ts`?